packages/twenty-front/src/modules/settings/lab/graphql/mutations/updateLabPublicFeatureFlag.ts

```diff
@@ -0,0 +1,15 @@
+import { gql } from '@apollo/client';
+
+export const UPDATE_LAB_PUBLIC_FEATURE_FLAG = gql`
+  mutation UpdateLabPublicFeatureFlag(
+    $workspaceId: String!
+    $publicFeatureFlag: String!
+    $value: Boolean!
+  ) {
+    updateLabPublicFeatureFlag(
+      workspaceId: $workspaceId
+      publicFeatureFlag: $publicFeatureFlag
+      value: $value
+    )
```

Comment:
Try to work with input typing, it is more robust to input updates
```suggestion
  mutation UpdateLabPublicFeatureFlag(
    $input: UpdateLabPublicFeatureFlagInput!
  ) {
    updateLabPublicFeatureFlag(data: $input)
```